package.json

```diff
@@ -27,27 +30,26 @@
   "dependencies": {
     "@actions/core": "^1.10.0",
     "@actions/exec": "^1.1.1",
-    "@actions/github": "^2.2.0",
-    "@octokit/webhooks": "^7.6.2",
-    "picomatch": "^2.2.2"
+    "@actions/github": "6.0.0",
+    "picomatch": "^2.3.1"
   },
   "devDependencies": {
-    "@types/jest": "^27.4.0",
-    "@types/js-yaml": "^3.12.4",
-    "@types/minimatch": "^3.0.3",
-    "@types/node": "^14.0.5",
-    "@types/picomatch": "^2.2.1",
-    "@typescript-eslint/eslint-plugin": "^5.10.2",
-    "@typescript-eslint/parser": "^5.10.2",
-    "@vercel/ncc": "^0.33.1",
-    "eslint": "^8.17.0",
-    "eslint-plugin-github": "^4.3.6",
-    "eslint-plugin-jest": "^22.21.0",
-    "jest": "^27.4.7",
-    "jest-circus": "^27.4.6",
-    "js-yaml": "^3.14.0",
-    "prettier": "^2.0.5",
-    "ts-jest": "^27.1.3",
-    "typescript": "^3.9.3"
+    "@octokit/webhooks-types": "^7.3.1",
+    "@types/jest": "^29.5.11",
+    "@types/js-yaml": "^4.0.9",
```

Comment:
4.0.9 does not exist => leading to 4.1.0
ref : https://www.npmjs.com/package/js-yaml?activeTab=versions (consulted on Feb 23, 2024)